assets/js/just-the-docs.js

```diff
@@ -145,6 +145,18 @@ function searchLoaded(index, docs) {
   var currentInput;
   var currentSearchIndex = 0;
 
+  {%- if site.search.focus_shortcut_key != false %}
```

Comment:
This check is buggy; `nil` and `false` are different in liquid (with respec to `==`), so this doesn't check if the key doesn't exist. 

(to test this: on this branch, remove the `focus_shortcut_key` you've added in `_config.yml`; observe that the following snippet still gets rendered, and the key is `''`)

The "falseyness" of `nil` is used when the expression itself is evaluated in a boolean context.

```suggestion
  {%- if site.search.focus_shortcut_key %}
```

Reply:
I made these changes in my initial commit, and I made sure to stick to the convention for other keys.

Agreed to your suggestion and made the changes. Thank you.